pagopa/io-app
# chore: [IOBP-695] Add banner of service to payments home screen

## ⚠️ This PR depends on https://github.com/pagopa/io-services-metadata/pull/787 ⚠️

## Short description
This PR adds the ability to display a remote alert (formally called a banner) within the home payment screen.

## List of changes proposed in this pull request
- Created a new `PaymentsAlertStatus` component that takes the `payments` section status to show/hide a banner;
- Updated io_content_specs definitions into package.json -> WARNING: As soon as the https://github.com/pagopa/io-services-metadata/pull/787 PR is merged, the content definition reference will have the 1.0.33 version
- Created a utility function `getAlertVariant` in order to "normalize" the alert variant based on the SectionStatus LevelEnum;

## How to test
- Checkout this PR from the dev-server: https://github.com/pagopa/io-dev-api-server/pull/389
- Run `yarn generate` both on the dev-server and the app
- Editing the `payments` attribute from the `backend.ts` file on the dev-server, you should be able to see or not see the alert.

## Preview

|Fade In|Fade Out|
|-|-|
|<video src="https://github.com/pagopa/io-app/assets/34343582/0c9506c3-da87-4087-bbaa-a29b16d8d822" />|<video src="https://github.com/pagopa/io-app/assets/34343582/6f7281d0-2d29-4653-883b-30548ffce959" />
